package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-callkeep",
-  "version": "4.1.0",
+  "version": "4.1.1-iosfix",
```

Comment:
Version bump so I can use this myself while we're reviewing. I assume we'd just change this when merging(?)